docs/install/tutorials/rpi.rst

```diff
@@ -179,7 +189,7 @@ You will need to write a configuration file with information about your local Wi
 
 .. code-block:: console
 
-  sudo nano /etc/default/hostapd
+  sudo nano /etc/hostapd/hostapd.conf
```

Comment:
Wrong file

Reply:
What version of Raspbian are you using?

Reply:
Buster -- but I think this was just a typo or mis-paste. It's the `DAEMON_CONF="/etc/hostapd/hostapd.conf"` line that goes in `/etc/default/hostapd`.

Reply:
Is it the desktop version Raspbian Buster or Buster Lite btw?

Reply:
I was using the Lite version.

Reply:
Cool! I was also using that version for a all the tests.